mottak-arkiv-web/src/components/workflow/QualityCheck.tsx

```diff
@@ -250,6 +263,8 @@ const QualityCheck: React.FC = (): JSX.Element => {
 								value={values.type}
 								onChange={handleValueChange}
 								fullWidth
+								helperText={validation['type'].hasError ? validation['type'].errorMsg : ''}
```

Comment:
```suggestion
								helperText={validation['type'].hasError ?? ''}
```
Hva skjer om du gjør det slik? Uten å teste koden, så trur jeg det skal fungere

Reply:
Vil ikke funke, se bort fra dette.